test/e2e/testdata/aemm-config-integ.json

```diff
@@ -10,7 +10,8 @@
   },
   "spot": {
     "action": "terminate",
-    "time": "2020-01-07T01:03:47Z"
+    "time": "2020-01-07T01:03:47Z",
+    "trigger-time": "2020-01-07T01:01:47Z"
```

Comment:
this shouldn't be needed here. If anything you could update this file with `mock-trigger-time` on the **root** level and consume this value in the e2e test. However, if you choose to pass in the value as a flag instead, then no change needed here.